homepage.css

```diff
@@ -0,0 +1,57 @@
+.quiz-container{
+    background-color:var(--bg-color);
+}
+.quiz-navbar-container{
+    flex-wrap: wrap;
+    font-size: 1rem;
+}
+.quiz-title{
+    text-transform: uppercase;
+}
+.intro-text{
+    line-height: 3rem;
+    text-align: left;
+    margin-left: 5rem;
+}
+.explore-btn{
+    display: block;
+    margin: 0;
+    margin-top: 1rem;
+}
+.quiz-img{
+    height: 20rem;
+    margin-right: 10rem;
+}
+.quiz-category-header{
+    font-size: xx-large;
+    font-weight: bolder;
+}
+.quiz-category-sections{
+    align-content: center;
+    flex-wrap: wrap;
```

Comment:
can create a class for flex-wrap and font-weight in component library and reuse it. Because you will come across it in many places.